lib/components/Modal/base/index.tsx

```diff
@@ -145,10 +145,12 @@ export const Modal = ({
                         />
                     )
                 ) : (
-                    <LabelPairedXmarkMdBoldIcon
+                    <button
                         className="quill-modal__close-icon"
                         onClick={toggleHandler}
-                    />
+                    >
+                        <LabelPairedXmarkMdBoldIcon fill="var(--component-textIcon-normal-prominent)" />
+                    </button>
```

Comment:
If we use our quill button component will have the padding I guess.

Reply:
we don't have this variant in quill button and it all comes with a border radius or underline. so i used custom button